lib/components/ArtistScreen/artist_item_info.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:get_it/get_it.dart';
+
+import '../../models/jellyfin_models.dart';
+import '../../screens/artist_screen.dart';
+import '../../services/jellyfin_api_helper.dart';
+import '../../services/process_artist.dart';
+import '../print_duration.dart';
+
+class ArtistItemInfo extends StatelessWidget {
+  const ArtistItemInfo({
+    Key? key,
+    required this.item,
+    required this.itemSongs,
+    required this.itemAlbums,
+  }) : super(key: key);
+
+  final BaseItemDto item;
+  final int itemSongs;
+  final int itemAlbums;
+
+// TODO: see if there's a way to expand this column to the row that it's in
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        _IconAndText(
+            iconData: Icons.music_note,
+            text: AppLocalizations.of(context)!.songCount(itemSongs)),
+        _IconAndText(
+            iconData: Icons.book,
+            text: AppLocalizations.of(context)!.albumCount(itemAlbums)),
+        if (item.genres != null && item.genres!.isNotEmpty)
+          _IconAndText(iconData: Icons.album, text: item.genres!.join(", "))
```

Comment:
This is the non-tappable version. Take a look at `ArtistIconAndText`, especially the used `GestureDetector` :)

Reply:
What'd happen if you click it and there are multiple genres?

Reply:
Right now it just goes to the first entry, it's not ideal. But we'll migrate it to chips (like the ones for artists) soon, that should work much better :)